docs/core/event_handler/bedrock_agents.md

```diff
@@ -0,0 +1,330 @@
+---
+title: Agents for Amazon Bedrock
+description: Core utility
+---
+
+Create [Agents for Amazon Bedrock](https://docs.aws.amazon.com/bedrock/latest/userguide/agents.html#agents-how){target="_blank"} using event handlers and auto generation of OpenAPI schemas.
+
+<center>
+```mermaid
+--8<-- "docs/core/event_handler/bedrock_agents.mermaid"
+```
+</center>
+
+## Key features
+
+* Minimal boilerplate to build Agents for Amazon Bedrock
+* Automatic generation of [OpenAPI schemas](https://www.openapis.org/){target="_blank"} from your business logic code
+* Built-in data validation for requests and responses
+* Similar experience when authoring [REST](api_gateway.md){target="_blank"} and [GraphQL APIs](appsync.md){target="_blank"}
+
+## Terminology
+
+**Data validation** automatically validates the user input and the response of your AWS Lambda function against a set of constraints defined by you.
+
+**Event handler** is a Powertools for AWS feature that processes an event, runs data parsing and validation, routes the request to a specific function, and returns a response to the caller in the proper format.
+
+**[OpenAPI schema](https://www.openapis.org/){target="_blank"}** is an industry standard JSON-serialized string that represents the structure and parameters of your API.
+
+**Action group** is a collection of two resources where you define the actions that the agent should carry out: an OpenAPI schema to define the APIs that the agent can invoke to carry out its tasks, and a Lambda function to execute those actions.
+
+**Large Language Models (LLM)** are very large deep learning models that are pre-trained on vast amounts of data, capable of extracting meanings from a sequence of text and understanding the relationship between words and phrases on it.
+
+**Agent for Amazon Bedrock** is an Amazon Bedrock feature to build and deploy conversational agents that can interact with your customers using Large Language Models (LLM) and AWS Lambda functions.
+
+## Getting started
+
+!!! tip "All examples shared in this documentation are available within the [project repository](https://github.com/aws-powertools/powertools-lambda-python/tree/develop/examples)"
+
+### Install
+
+!!! info "This is unnecessary if you're installing Powertools for AWS Lambda (Python) via [Lambda Layer/SAR](../../index.md#lambda-layer){target="_blank"}."
+
+You need to add `pydantic` as a dependency in your preferred tool _e.g., requirements.txt, pyproject.toml_. At this time, we support both Pydantic V1 and V2. For a future major version, we will only support Pydantic V2.
```

Comment:
Hey @rubenfonseca, please correct if I'm wrong, but Agents for Bedrock doesn't support OpenAPI 3.1+ and Pydantic v2 is only compatible with Pydatinc v2, right? Because of that we fixed this problems in this PR: https://github.com/aws-powertools/powertools-lambda-python/pull/3860

Reply:
Nice catch, changed the sentence. Let me know if it's ok